migration/sql-files/010-add-cluster-to-user.sql

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE users ADD COLUMN IF NOT EXISTS cluster TEXT;
+UPDATE users SET cluster = '{{ index . 0}}'  WHERE cluster is null or cluster = '';
+ALTER TABLE users ALTER COLUMN cluster set NOT NULL ,ADD CHECK (cluster <> '');
```

Comment:
Hi @sbryzak 
Do you seeing anything obvious which I should be taking care of , in my migration scripts to ensure that concurrent migration tests don't fail ? I see failure at the moment.. [1] 

Here are my database changes :  https://github.com/fabric8-services/fabric8-auth/pull/164/files#diff-7821b63d24bdd94b4c4cc05c4ff485c8

Here's the test that fails:
https://github.com/fabric8-services/fabric8-auth/blob/master/migration/migration_test.go

[1] https://ci.centos.org/job/devtools-fabric8-auth/362/console

Reply:
I sent you some direct messages in Mattermost but it seems you're offline, anyway to repeat:

1. The DDL syntax looks to be only valid for postgresql 9.6 and higher: https://stackoverflow.com/questions/12597465/how-to-add-column-if-not-exists-on-postgresql

2. It looks like we are using Postgresql 9.5: 
(from the Jenkins log)
Pulling postgres_integration_test (centos/postgresql-95-centos7:latest)...

3. To fix the problem we could try updating to this postgres image instead:

https://hub.docker.com/r/centos/postgresql-96-centos7/

Reply:
@sbryzak Thanks a ton!
You were absolutely right. I removed the `IF NOT EXISTS` and it worked :)
since updating the version might not be an option.

Probably we should update the local postgres version to match it with ci.centos.